eradio.el

```diff
@@ -46,13 +46,29 @@ This is a list of the program and its arguments.  The url will be appended to th
 
 (defvar eradio-process nil "The process running the radio player.")
 
+(defvar eradio-current-channel nil "Currently playing (or paused) channel.")
+
 (defun eradio-alist-keys (alist)
   "Get the keys from an ALIST."
   (mapcar #'car alist))
 
 (defun eradio-stop ()
   "Stop the radio player."
-  (interactive) (when eradio-process (delete-process eradio-process)))
+  (interactive)
+  (when eradio-process
+    (delete-process eradio-process)
+    (setq eradio-current-channel nil)
+    (setq eradio-process nil)))
```

Comment:
Setting eradio-process to `nil` after it has been deleted is redundant, no?

Reply:
my bad

Reply:
I was thinking stop completely clears everything so that after calling `stop` if user calls `toggle` then they can select radio channel. But it's a matter of taste I guess.

Reply:
I think `eradio-play`/`eradio-toggle` is more efficient, since every action (play previous channel, pick a channel to play and stop player) can be performed with a single command.